tests/sentry/api/endpoints/test_rule_snooze.py

```diff
@@ -601,8 +665,8 @@ def test_delete_metric_alert_rule_mute_everyone(self):
         assert response.status_code == 204
 
     @with_feature("organizations:mute-alerts")
-    def test_cant_delete_metric_alert_rule_mute_everyone(self):
-        """Test that a user can't unsnooze a metric rule that's snoozed for everyone if they do not belong to the team the owns the rule"""
+    def test_can_delete_metric_alert_rule_mute_everyone(self):
+        """Test that a user can unsnooze a metric rule that's snoozed for everyone if they do not belong to the team the owns the rule"""
         other_team = self.create_team()
```

Comment:
@roggenkemper Hmm why do we create the team if the team doesn't matter anymore? I also don't think we need new tests, I'd think we'd just repurpose `test_cant_delete_metric_alert_rule_mute_everyone` to `test_delete_metric_alert_rule_without_alert_write`